### PR TITLE
[IMP] ribbon color customization

### PR DESCRIPTION
**Pull request by jabibi**
## web_environment_ribbon

The ribbon color and background-color can be modified through system parameters:
- ribbon.color
- ribbon.background.color

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ addon | version | summary
 [web_dialog_size](web_dialog_size/) | 8.0.0.1.0 | A module that lets the user expand a dialog box to the full screen width.
 [web_dom_model_classes](web_dom_model_classes/) | 8.0.1.0.0 | Allows small UI changes with simple CSS
 [web_easy_switch_company](web_easy_switch_company/) | 8.0.1.0.0 | Multicompany - Easy Switch Company
-[web_environment_ribbon](web_environment_ribbon/) | 8.0.0.1.0 | Web Environment Ribbon
+[web_environment_ribbon](web_environment_ribbon/) | 8.0.1.0.0 | Web Environment Ribbon
```
Comment (pedrobaeza) [inline README.md]: Don't change this. It's automatically changed when this module gets merged.

Comment (pedrobaeza) [inline README.md]: You have to clean your commits to remove the diff in this file, or there will be conflicts for sure.

Comment (jabibi) [inline README.md]: Ok, I didn't know

**Comment by pedrobaeza** [comment]
Write here when you finish to review full PR. Thanks for the contribution!

**Comment by jabibi** [comment]
Just done it
Hope this will be usefull

File: web_environment_ribbon/README.rst [hunk 2]
```diff
@@ -12,46 +12,41 @@ Mark a Test Environment with a red ribbon on the top left corner in every page
...
-   the default system parameter "ribbon.name"
-   (in the menu Settings > Parameters > System Parameters)
-   To hide the ribbon, set this parameter to "False" or
-   delete it.
+* You can change the ribbon's name ("TEST") by editing the default system
+  parameter "ribbon.name" (in the menu Settings > Parameters > System
+  Parameters) To hide the ribbon, set this parameter to "False" or delete it.
+* You can customize the ribbon color and background color through system
+  parameters: "ribbon.color", "ribbon.background.color". Fill with valid CSS
+  colors or just set to "False" to use default values.
```
Comment (andhit-r) [inline web_environment_ribbon/README.rst:25]: can we use hex notation for ribbon.background.color?

Comment (jabibi) [inline web_environment_ribbon/README.rst:25]: Yes, all CSS values for colors are accepted

**Comment by andhit-r** [comment]
Tested on runbot :+1:

**Comment by pedrobaeza** [comment]
Tested also in runbot and working :+1:

**Comment by lfreeke** [comment]
Tested on runbot :+1:

**Pull request merged**